README.md

```diff
@@ -72,6 +72,25 @@ Enums.Framerate framerate = timecodeObj.Framerate;
 string timecodeRegex = Timecode.RegexPattern;
 ```
 
+### Operator Overloading
+
+```csharp
+
+var tc1 = new Timecode(1, 2, 20, 100, DotnetTimecode.Enums.Framerate.fps23_976);
+var tc2 = new Timecode(0, 22, 31, 100, DotnetTimecode.Enums.Framerate.fps23_976);
+
+
+Console.WriteLine(tc1 + tc2); //01:24:59:08
+Console.WriteLine(tc1 - tc2); //00:39:49:00
```

Comment:
As we will change the the values of tc1 and tc2, the comments here should also be updated.

Reply:
Done